features/google/src/components/dataStructureMiniStory/dataStructureMiniStory.css

```diff
@@ -0,0 +1,3 @@
+.data-structure-source {
```

Comment:
Was this PR meant to add a bunch of UI code?

Reply:
Nope, I pushed to the wrong branch 😅